src/test/resources/com/puppycrawl/tools/checkstyle/main/config-custom-root-module.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
```

Comment:
In latest master, only `MainTest` uses this file, so it should be moved.
Please rebase on latest master if you aren't already.